pkg/controllers/mcs/service_export_controller.go

```diff
@@ -81,7 +81,7 @@ func (c *ServiceExportController) Reconcile(ctx context.Context, req controllerr
 		return controllerruntime.Result{}, nil
 	}
 
-	if !helper.IsWorkContains(&work.Status, serviceExportGVK) {
```

Comment:
> Therefore, the work status of the ServiceExport type should not be collected during work status collection. 

Do you mean here `helper.IsWorkContains` always return `false`? 

By the way, should we add `ServiceExport`/`ServiceImport` to interpreter framework?

Reply:
> Do you mean here helper.IsWorkContains always return false?

For now, even if the resource doesn't have status, we still will collect resource `ManifestStatuses.ResourceIdentifier`, `helper.IsWorkContains` is judging by  `ManifestStatuses.ResourceIdentifier`.